pkg/dtos/keydata.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2024 IOTech Ltd
+
+package dtos
+
+import (
+	"strings"
+
+	"github.com/IOTechSystems/go-mod-central-ext/v4/pkg/models"
+)
+
+type KeyData struct {
+	Issuer string `json:"issuer" validate:"required"`
+	Type   string `json:"type" validate:"omitempty,oneof='verification' 'signing'"`
```

Comment:
The single quote is not necessary unless the value contains space. Probably we can remove them.
```suggestion
	Type   string `json:"type" validate:"omitempty,oneof=verification signing"`
```

Reply:
Removed.